Extensions/CMake/SlicerFunctionGenerateExtensionDescription.cmake

```diff
@@ -193,7 +193,7 @@ function(slicer_generate_extension_description_test)
     EXTENSION_STATUS ""
     )
   execute_process(
-    COMMAND ${CMAKE_COMMAND} -E compare_files
+    COMMAND ${CMAKE_COMMAND} -E compare_files --ignore-eol
```

Comment:
Is this option available with the minimum required version of CMake associated with Slicer ?

Reply:
I assumed that it is, I have not checked it.